test/common/model/fields.spec.js

```diff
@@ -1,90 +0,0 @@
-const chai = require('chai');
```

Comment:
This test is never run. That source file is also never used, so I should delete it, too.